zeebe/broker/src/test/java/io/camunda/zeebe/broker/exporter/stream/ExporterDirectorTest.java

```diff
@@ -664,6 +664,29 @@ public void shouldRecoverFromStartWithNonUpdatingExporter() throws Exception {
     assertThat(exporters.get(1).getExportedRecords()).isEmpty();
   }
 
+  @Test
+  public void shouldStartContainersSoftPaused() {
+    // All containers of an exporter should be initialized as soft paused if this is also
+    // the exporter director phase on initialization. This is to prevent that the
+    // that exporter containers can start to update the exported position before the exporter
+    // director is ready. When soft paused new events are exported the position should not update.
+
+    // when
+    rule.startExporterDirector(exporterDescriptors, ExporterPhase.SOFT_PAUSED);
```

Comment:
:x: I just realized that the exporters are no updating the position by default. You have to set it explicitily:
```
exporters.get(0).shouldAutoUpdatePosition(true);
exporters.get(1).shouldAutoUpdatePosition(true);
```
Otherwise, the test passes even with the bug.  May be verify if the test can reproduce the bug with out the fix.

Reply:
Thanks good catch!